app/helpers/qunit_helper.rb

```diff
@@ -1,6 +1,35 @@
 # frozen_string_literal: true
 
 module QunitHelper
+
+  def support_bundles
+    result = []
+    if Rails.env.production? || @legacy_ember
+      result << preload_script("discourse/tests/test-support-rails")
+      result << preload_script("discourse/tests/test-helpers-rails")
+    else
+      result << preload_script("test-support")
+      result << preload_script("test-helpers")
+    end
+    result.join("\n").html_safe
+  end
+
+  def boot_bundles
+    result = []
+    if @legacy_ember
+      result << preload_script("discourse/tests/test_starter")
+    else
+      if @is_proxied
+        result << preload_script("scripts/discourse-test-listen-boot")
+        result << preload_script("scripts/discourse-boot")
+      else
+        result << preload_script("discourse-test-listen-boot")
+        result << preload_script("discourse-boot")
+      end
```

Comment:
So far so good! Just a minor readability suggestion:

```suggestion
    elsif @is_proxied
      result << preload_script("scripts/discourse-test-listen-boot")
      result << preload_script("scripts/discourse-boot")
    else
      result << preload_script("discourse-test-listen-boot")
      result << preload_script("discourse-boot")
```